echion/frame.h

```diff
@@ -116,10 +119,16 @@ class Frame
     Frame(PyObject *frame)
     {
 #if PY_VERSION_HEX >= 0x030b0000
+
+#if PY_VERSION_HEX >= 0x030d0000
+        _PyInterpreterFrame* iframe = (_PyInterpreterFrame *)frame;
+        const int lasti = _PyInterpreterFrame_LASTI(iframe);
```

Comment:
I'm not sure why it's ok to call `_PyInterpreterFrame_LASTI` here and not in line 403.